cms/envs/common.py

```diff
@@ -208,6 +208,12 @@
 
     # Show Language selector
     'SHOW_LANGUAGE_SELECTOR': False,
+
+    # Temporary feature flag for disabling saving of subsection grades.
+    # There is also an advanced setting in the course module.  This
+    # feature flag is used only to toggle whether the advanced setting
+    # appears in Studio.
+    'ENABLE_SUBSECTION_GRADES_SAVED': False,
```

Comment:
I think I agree with @efischer19 on this.  If something goes wrong, and we need to kill the feature after we deploy it (say a performance issue), we need the ability to turn off robust grading for courses that have enabled it, not just to turn off the ability for course teams to enable it on new course.  So having the feature flag/advanced setting redundancy would be useful.